frontend/src/pages/PageViews.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+    <ContentLayout :title="title">
+        <VLayout
+            wrap
+        />
+        <VLayout>
+            <VFlex
+                lg12
+                md12
+                sm12
+                xs12
+                class="content-card"
+            >
+                <VLayout
+                    wrap
+                    align-center
+                    justify-center
+                >
+                    <VFlex
+                        class="chart-container"
+                    >
+                        <LineChart :data="data" />
+                        <PeriodDropdown />
+                    </VFlex>
+                </VLayout>
+            </VFlex>
+        </VLayout>
+        <VLayout class="buttons-row">
+            <VFlex
+                xs12
+                sm8
+                offset-sm2
+            >
+                <VLayout>
+                    <VFlex
+                        v-for="button in buttons"
+                        :key="button.title"
+                    >
+                        <ButtonComponent
+                            :title="button.title"
+                            :type="button.type"
+                            :icon-name="button.icon"
+                        />
+                    </VFlex>
+                </VLayout>
+            </VFlex>
+        </VLayout>
+        <VLayout>
+            <VFlex
+                lg12
+                md12
+                hidden-sm-and-down
+                height="100%"
+                class="img-card"
+            >
+                <GroupedTable
+                    :items="tableData"
+                />
+            </VFlex>
+        </VLayout>
+    </ContentLayout>
+</template>
+
+<script>
+    import ContentLayout from '../components/layout/ContentLayout.vue';
+    import LineChart from "../components/common/LineChart";
+    import GroupedTable from "../components/dashboard/page_views/GroupedTable";
+    import ButtonComponent from "../components/dashboard/page_views/ButtonComponent";
+    import PeriodDropdown from "../components/dashboard/page_views/PeriodDropdown";
+    import {isWebsite} from '../mixins/isWebsite';
+    import {
+        PAGE_VIEWS,
+        UNIQUE_PAGE_VIEWS,
+        ACTIVE_USERS,
+        BOUNCE_RATE
+    } from '../configs/page_views/buttonTypes.js';
+
+    export default {
+        mixins: [isWebsite],
```

Comment:
don't put guard on each child page, you have 1 page that wraps all of them, which means that guard mixin should be used only there (it also means that Visitors page no longer need it too)